### PR TITLE
EPH WebSocket Support

### PR DESCRIPTION
**Pull request by tiny-dancer**
Enhancement

add `eph` support for websocket connections with `eph.WithWebSocketConnection()`

- [x] All tests passed
- [x] Add change to `changelog.md`

### Environment
- OS: Write here
- Go version: Write here

### CODE REVIEW

File: eph/eph.go [hunk 1]
```diff
@@ -117,6 +118,14 @@ func WithEnvironment(env azure.Environment) EventProcessorHostOption {
 	}
 }
 
+// WithEnvironment will configure an EventProcessorHost to use the specified Azure Environment
```
Comment (gavinfish) [inline eph/eph.go]: Misleading comment.

**Comment by tiny-dancer** [comment]
@gavinfish updated descriptions and changelog.  thought about this change allowing the caller to pass in `[]HubOpts` from the `eph.New()` commands however opted for the simpler already existing pattern.

unfortunately, this one falls under company-based contribution so CLA may take a few days, will sign once good on our end 👍 

Thanks for the quick review!

**Comment by tiny-dancer** [comment]
@gavinfish all good with the CLA

**Comment by tiny-dancer** [comment]
hey all! I see all checks have passed - is this waiting on anything to be merged? Thanks.

**Comment by jhendrixMSFT** [comment]
Sorry for the delay.

Can you please update [version.go](https://github.com/Azure/azure-event-hubs-go/blob/master/version.go) to v3.1.0?  Once that's done I can merge and tag it.

**Comment by tiny-dancer** [comment]
@jhendrixMSFT boom, good to go! no worries on delay.

**Pull request merged**